commands/azureCommands/acr-build.ts

```diff
@@ -0,0 +1,115 @@
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry/lib/containerRegistryManagementClient';
```

Comment:
One more bigger work item for you:  Please go to tsconfig.json and turn on "strictNullChecks", then fix up all errors in the new code (I've already done work to get rid of a lot of them in the rest of the repo, so we want new code to be clean).  This oughtta be a separate commit. We can discuss it coming later if you want, but it catches stuff like using ext.ui.showQuickPick (which throws on cancel) instead of vscode.window.showQuickPick (which returns undefined on cancel, and which caller may not handle properly).

You'll have to turn it off again afterwards.

Reply:
Great tip!
Are there any others I should temporarily turn on?